### PR TITLE
Css fixes and relayout

### PR DESCRIPTION
**Pull request by galczo5**
localhost_8000_jazda_ (5)
localhost_8000_jazda_ (6)

### CODE REVIEW

**Comment by galczo5** [comment]
localhost_8000_jazda_ (7)

File: house_of_refuge/frontend/src/components/ResourceList.js [hunk 1]
```diff
@@ -198,12 +200,14 @@ export const ResourceList = ({initialResources, sub, subHandler, user, clearActi
...
-            <input className="search-input" onChange={(e) => setSearchQuery(e.target.value.toLowerCase())}/>
+          <QuickFilter label={"Szukaj"}>
+            <div className={"d-flex align-items-center"}>
+              <Search/>
+              <input className="search-input" onChange={(e) => setSearchQuery(e.target.value.toLowerCase())}/>
+            </div>
           </QuickFilter>
         </div>
-        <div className={"d-flex justify-content-around"}>
+        <div className={"d-flex justify-content-between align-items-center mx-5"}>
```
Comment (tadeoos) [inline house_of_refuge/frontend/src/components/ResourceList.js:210]: this one i'm not sure about.. kinda liked it wasn't jsutified to the left.. It seems less noticeable here, wdyt about brining it closer to the pagination? Maybe it should actually be right beside it?

File: house_of_refuge/frontend/src/components/SubmissionRow.js [hunk 2]
```diff
@@ -150,8 +150,14 @@ export function SubmissionRow({sub, activeHandler, user, isGroupCoordinator, isA
   return <div
       className={`submission-row ${getStatusClass(localSub)}
       ${localSub.accomodation_in_the_future ? "sub-in-future" : ""} ${isActive ? "sub-active" : ""}`}>
-    <p className="sub-id">ID ZGŁOSZENIA: {localSub.id}</p>
-    <Table className="sub-table">
+    <div className="sub-id position-relative">
+      ID ZGŁOSZENIA: {localSub.id}
+
+      <div className="submission-row-collapse" onClick={() => setCollapsed(!collapsed)}>
```
Comment (tadeoos) [inline house_of_refuge/frontend/src/components/SubmissionRow.js:156]: add `cursor: pointer` here

File: house_of_refuge/frontend/src/components/SubmissionRow.js [hunk 3]
```diff
@@ -150,8 +150,14 @@ export function SubmissionRow({sub, activeHandler, user, isGroupCoordinator, isA
...
+        Zwiń / Rozwiń
+      </div>
+    </div>
+    <Table className="sub-table" style={{'background-color': 'rgba(255, 255, 255, 0.9)', display: collapsed ? 'none' : 'table'}}>
```
Comment (tadeoos) [inline house_of_refuge/frontend/src/components/SubmissionRow.js:160]: I liked a bit more of contrast maybe bump 0.9 -> 0.95 ?

File: house_of_refuge/frontend/src/scripts/index.js [hunk 4]
```diff
@@ -21,17 +21,18 @@ import {BrowserRouter, Route, Routes, useSearchParams} from "react-router-dom";
...
   useInterval(async () => {
     const newHelped = await getHelped();
     setPeopleHelped(newHelped);
   }, 120 * 1000);
 
-  return <div className="panel-header" style={hide ? {display: "none"} : {}}>
+  return <div className="panel-header mx-5 mt-1" style={hide ? {display: "none"} : {}}>
     <div>
-      <img src="/static/images/logo.svg" alt="logo" style={{height: "76px", margin: "10px 0"}}/>
+      <img src="/static/images/logo.svg" alt="logo" style={{height: "76px", margin: "10px 0", display: collapsed ? 'none' : 'block'}}/>
```
Comment (tadeoos) [inline house_of_refuge/frontend/src/scripts/index.js:33]: should we be reusing/combining the hide and collapses params here?

File: house_of_refuge/frontend/src/styles/resources.scss [hunk 5]
```diff
@@ -102,10 +102,11 @@ a {
 }
 
 .sub-table {
-  background-color: white;
   font-size: 12px;
   border-radius: 0;
+  table-layout: fixed;
 
+  background:
```
Comment (tadeoos) [inline house_of_refuge/frontend/src/styles/resources.scss:109]: residue?

File: house_of_refuge/frontend/src/styles/resources.scss [hunk 6]
```diff
@@ -452,17 +470,49 @@ a {
...
   label {
     margin-right: 6px;
   }
+
+  &__children {
+    height: 34px;
+    display: flex;
+
+    > div {
+      width: 100%;
```
Comment (tadeoos) [inline house_of_refuge/frontend/src/styles/resources.scss:496]: the switch buttons seem a bit heavy to me when they have full width, but maybe it's better to be consistent.. ideally I'd add some max-width to them

**Review by tadeoos — CHANGES_REQUESTED** [review]
great work! Only a couple of small stuff left

File: house_of_refuge/frontend/src/components/SubmissionRow.js (hunk 3 above)
Comment (galczo5) [reply house_of_refuge/frontend/src/components/SubmissionRow.js:160]: Done

File: house_of_refuge/frontend/src/styles/resources.scss (hunk 5 above)
Comment (galczo5) [reply house_of_refuge/frontend/src/styles/resources.scss:109]: done

File: house_of_refuge/frontend/src/styles/resources.scss (hunk 6 above)
Comment (galczo5) [reply house_of_refuge/frontend/src/styles/resources.scss:496]: Should be easier to use on mobile devices. I'll set a reasonable max width

File: house_of_refuge/frontend/src/components/ResourceList.js (hunk 1 above)
Comment (galczo5) [reply house_of_refuge/frontend/src/components/ResourceList.js:210]: Moved it to flex end 
image

**Pull request merged**